### PR TITLE
recipe App

### PR DESCRIPTION
**Pull request by Richez003**
The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them...
Also. I was working alone this week on the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+
+- GitHub: [@Richez003](https://github.com/Richez003)
+- LinkedIn: [Richard Friday](https://www.linkedin.com/in/richard-friday)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Richez003/Recipe-app-RailsGroupProject/issues).
```
Comment (mikethreels) [inline README.md]: - [x] this link is still not working as requested by the previous CR, please fix it.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Richard, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

### Please do not create a new PR when requesting for a re-review.

 ## Highlights
- good job following the instructions from the previous CR

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Richez003) [reply README.md]: done . thanks

**Review by youmari — APPROVED** [review]
Hi @Richez003 , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**